src/util/string.cpp

```diff
@@ -574,6 +574,20 @@ bool parseColorString(const std::string &value, video::SColor &color, bool quiet
 	return success;
 }
 
+std::string encodeHexColorString(const video::SColor &color)
```

Comment:
If you were to switch this from colorstrings to colorspecs, you could use `push_ARGB8` to push a colorspec (table with a, r, g, b fields) here.

Reply:
Done, keeping this method for item meta json serialization